### PR TITLE
docs: Release notes

### PR DESCRIPTION
**Pull request by Jimlinz**
## Reference

[Code review checklist](https://github.com/linz/geostore/blob/master/CODING.md#Checklist)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,6 +350,29 @@ limit the commands to the specific dataset!_
...
 
+## Geostore Release
+
+### Versioning and Frequency
+
+We aim to release at the end of each agile sprint (fortnightly), or whenever required (e.g. bugfix,
+feature rollout).
+
+Geostore follows [semantic versioning](http://semver.org/). Each release is incrementally tag with
+`release-x.yy.z` (where x.yy.z denotes a version number).
```
Comment (l0b0) [inline README.md:361]: As per discussion, maybe something like "`release-major.minor.patch` (for example, `release-12.10.2`)"?

File: README.md [hunk 2]
```diff
@@ -350,6 +350,28 @@ limit the commands to the specific dataset!_
...
+commit titles since the last release. Commits from dependabot are excluded from automatically
+generated release notes, as specified in `.github/release.yml`. You should always check the release
+notes and update accordingly as needed.
+
+Note: Geostore has no rollback process. Any fixes will need to be carried out in a roll forward
+basis.
+
 [^1]:
 
 When using Nix, make sure to remove the `.venv` directory. Mixing Nix and Poetry leads to weird
```
Comment (billgeo) [inline README.md:377]: Is this line related to release process? Or is this a not for developers or something?

File: README.md [hunk 3]
```diff
@@ -350,6 +350,28 @@ limit the commands to the specific dataset!_
    1. Ask AWS support to remove the delete markers returned by
       `aws s3api list-object-versions --bucket=linz-geostore --prefix=DATASET_TITLE/ | jq .DeleteMarkers`.
 
+## Geostore Release
+
+### Versioning and Frequency
+
+We aim to release at the end of each agile sprint (fortnightly), or whenever required (e.g. bugfix,
```
Comment (billgeo) [inline README.md:357]: Do we need to mention that publishing a release triggers a prod deployment via GH actions?

File: README.md (hunk 2 above)
Comment (billgeo) [reply README.md:377]: Oh right. hadn't seen that notation before. :smi

Comment (Jimlinz) [reply README.md:377]: This was fixed in https://github.com/linz/geostore/pull/2180, just need to update the branch.

**Review by billgeo — APPROVED** [review]
